docs/reference/api/docker-io_api.md

```diff
@@ -10,502 +10,5 @@ parent = "smn_remoteapi"
 
 # Docker Hub API
 
-- This is the REST API for [Docker Hub](https://hub.docker.com).
-- Authorization is done with basic auth over SSL
-- Not all commands require authentication, only those noted as such.
+This API is deprecated as of 1.7. You should use the [Docker Registry HTTP API V2](https://docs.docker.com/registry/spec/api/).
```

Comment:
this is actually a different API from the registry api. This is the Hub 1.0 API's which haven't been replaced yet.